src/UKHO.ERPFacade.API/Controllers/WebhookController.cs

```diff
@@ -21,6 +21,7 @@ public class WebhookController : BaseController<WebhookController>
         private readonly IXmlHelper _xmlHelper;
 
         public const string TRACEIDKEY = "data.traceId";
+        private const string UPDATE_REQUEST_TIME = "RequestDateTime";
```

Comment:
Should be UpdateRequestTime

Reply:
Removed the constant as it is no longer needed